docs/src/_contribute/cli.md

```diff
@@ -0,0 +1,225 @@
+
+The `--format` option should still default to `text` unless the user explicitly specifies a format. The two primary
+formats are:
+
+- `text`: human-readable plain text output.
+- `json`: JSON output.
+
+In the future we may add support for other formats (e.g. `yaml`).
+
+### Expected help and usage
+
+Feature groups should have fully documented help and usage, that contains at least basic invocation examples. And link
+to the CLI documentation for that specific feature group for more details.
+
+```
+Usage: meltano job [OPTIONS] COMMAND [ARGS]...
+
+  Manage jobs.
+
+  Example usage:
+
+      # This help
+      meltano job --help
+      # List all jobs in JSON format
+      meltano job list --format json
+      # List a named job
+      meltano job list [JOB_NAME]
+
+      # Create a new job with a single task representing a single run command.
+      meltano job add NAME --tasks 'tap mapper target command:arg1'
+
+      # Create a new job with multiple tasks each representing a run command.
+      # The list of tasks must be yaml formatted and consist of a list of strings, list of string lists, or mix of both.
+      meltano job add NAME --tasks '["tap mapper target", "tap2 target2", ...]'
+      meltano job add NAME --tasks '[["tap target dbt:run", "tap2 target2", ...], ...]'
+
+      # Remove a named job
+      meltano job remove NAME
+
+ Read more at https://docs.meltano.com/reference/command-line-interface#jobs
+
+Options:
+  --database-uri TEXT  System database URI.
+  --help               Show this message and exit.
+
+Commands:
+  add     Add a new job with tasks.
+  list    List job(s).
+  remove  Remove a job.
+  set     Update an existing jobs tasks
+```
+
+### Help style guidelines
+
+#### Required items
+
+For required items such as commands and arguments, use text without brackets or braces. In the following examples, all
+words and arguments are required:
+
+```
+meltano run JOB_NAME
+meltano invoke PLUGIN_NAME
+meltano schedule list
+meltano schedule remove SCHEDULE_NAME
+```
+
+#### Optional items
+
+Use square brackets around an optional items. If there's more than one optional item, enclose each item in
+its own set of square brackets. In the following example the `--log-level` and `--dump` are optional, while PLUGIN_NAME
+is required:
+
+```
+meltano [--log-level=LEVEL] invoke [--dump=config] PLUGIN_NAME
+```
+
+#### Mutually exclusive items
+
+Use curly braces (`{}`) to indicate that the user must choose one—and only one—of the items inside the braces. Use
+PIPES (`|`) to separate the items:
+
+```
+meltano schedule {list|remove}
+```
+
+#### Repeating items
+
+Use three trailing dots and no spaces (`...`) to indicate that the user can specify multiple values for the items:
+
+```
+meltano run BLOCKS ...
+meltano install [] [PLUGIN_NAME] ...
+```
+
+### Phrasing guidelines for command Deprecation and Preview commands
```

Comment:
@tayloramurphy and specific call outs or considerations for how to handle preview commands or deprecating commands?

Reply:
@pandemicsyn only thing that comes to mind is recommending that we call out which major version number we expect to promote or deprecate a preview command.

Reply:
👍 thats a good call out.